src/sfctl/tests/help_text_test.py

```diff
@@ -0,0 +1,364 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Tests to ensure that -h does not return error when run. Also ensures that help
+text exists. This only tests for commands/subgroups as specified by the user."""
+
+from __future__ import print_function
+import unittest
+from sys import stderr
+from subprocess import (Popen, PIPE)
+
+class HelpTextTests(unittest.TestCase):
+    """Tests which ensure that help texts do not error out and includes help text."""
+
+    def _validate_output_validate_line(self, #pylint: disable=too-many-arguments, inconsistent-return-statements
```

Comment:
I don't understand the returns here, if the function is validating something it seems like it should return a boolean. Maybe rename this so the behavior is more obvious.